README.md

```diff
@@ -15,4 +15,8 @@ results you would get.
 Also give a theoretical argument for why X could or could not be correct, based
 on the complexity of the general sorting problem we covered in class.
 
+I would start out a practical test of the algorithms runtime by trying it on edge case lists, and common worst case scenario lists: reverse sorted, nearly sorted but with last element in first index, completely random, increasingly large data sets, lists with repeat elements, lists of different data types (floats, doubles, etc.), etc. I would also do intensive testing over a large set of lists, all random, to see if the run time continues to hold. I would also see how it compares to other common or fast algorithms in similar circumstances to see what data it excels on and what it struggles on to further be able to test it. Given my skepticism below I would expect to see that when testing with a large random data set, which includes edge case tests, the average runtime would level out to a more reasonable nlogn or worse. Comparing to other algorithms I would likely see that it levels out with them and there would be algorithms that test similarly on the same edge cases, and be able to determine at a base level how the sorting algorithm works. You would see how the measured runtime on whatever worst case scenario you find changed as you use larger data sets, as well as how the average changes with larger data sets, and how best case scenario changes with larger data sets. Which should show growth rates that can be corresponded to particular complexities.
```

Comment:
How would you tell whether the complexity is linear?

Reply:
If the accumulation of edge cases, random cases, and excessively large data sets resulted in a time linear to the size of the inputs